code/game/objects/effects/spawners/random/structure.dm

```diff
@@ -25,17 +25,19 @@
 /obj/effect/spawner/random/structure/girder
 	name = "girder spawner"
 	icon_state = "girder"
+	spawn_loot_chance = 90                   // 10% chance of nothing
 	loot = list(
-		/obj/structure/girder = 4,
-		/obj/structure/girder/displaced = 1,
+		/obj/structure/girder = 8,           // 80% chance
```

Comment:
Alignment like this is banned. Don't do it.

```suggestion
		/obj/structure/girder = 8, // 80% chance
```

Reply:
Here and the others.

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/65252/commits/1cd84c4170d9bb5f4a24056d99daa2b37b387edb